keps/sig-network/2200-externalips-admission/kep.yaml

```diff
@@ -0,0 +1,43 @@
+title: Block ExternalIPs via Admission Control
+kep-number: 2200
+authors:
+  - "@thockin"
+owning-sig: sig-network
+participating-sigs:
+  - sig-auth
+  - sig-security
+  - sig-api-machinery
+status: implementable
```

Comment:
I have taken the liberty of marking it implementable because https://github.com/kubernetes/kubernetes/pull/97395 exists

@tallclair PTAL